src/Cli/dotnet/NugetSearch/NugetToolSearchApiRequest.cs

```diff
@@ -23,8 +23,8 @@ public async Task<string> GetResult(NugetSearchApiParameter nugetSearchApiParame
                 nugetSearchApiParameter.Prerelease);
 
             var httpClient = new HttpClient();
-            var cancellation = new CancellationTokenSource(TimeSpan.FromSeconds(10));
-            HttpResponseMessage response = await httpClient.GetAsync(queryUrl, cancellation.Token);
+            using var cancellation = new CancellationTokenSource(TimeSpan.FromSeconds(10));
+            HttpResponseMessage response = await httpClient.GetAsync(queryUrl, cancellation.Token).ConfigureAwait(false);
```

Comment:
💭 It does not appear that this file uses explicit `ConfigureAwait` calls.

Reply:
After a second review, I even think it's not recommended to use ConfigureAwait(false) here.
Thanks for review, I'll revert it.